### ISSUE TITLE
Provide the bazel rule for thrift py using .thirft file

### ISSUE THREAD

**Issue by SidduMirji** [twitter/heron#1942]

**Comment by kramasamy** [comment]
@SidduMirji - we currently do not use thrift at all in Heron. Hence we have not added any bazel rule for thrift.

**Comment by maosongfu** [comment]
@kramasamy Not really. We are using thrift for ScrbieSink in MetricsMgr: https://github.com/twitter/heron/tree/master/heron/metricsmgr/src/thrift

**Comment by kramasamy** [comment]
Sorry! I did not realize it.
